services-directions/src/test/java/com/mapbox/api/directions/v5/models/WalkingOptionsTest.java

```diff
@@ -0,0 +1,46 @@
+package com.mapbox.api.directions.v5.models;
+
+  import com.mapbox.api.directions.v5.WalkingOptions;
+  import com.mapbox.core.TestUtils;
+
+  import org.junit.Test;
+
+  import static org.junit.Assert.assertEquals;
+
+public class WalkingOptionsTest extends TestUtils {
+ private static final String JSON = "{\"walkingSpeed\":1.0,\"walkwayBias\":2.0,\"alleyBias\":3.0}";
```

Comment:
This JSON isn’t something the backend would return they would return snake_case except for `maxHikingDifficulty` - wondering if we could ask to services team to make it snake_case too for consistency 🤔

Reply:
I've just 👀 that `maxHikingDifficulty` is not supported anymore and edited ☝️